app/proguard-rules.pro

```diff
@@ -1,6 +1,6 @@
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in /home/the8ball/Android/Sdk/tools/proguard/proguard-android.txt
+# in /Users/creyes/Library/Android/sdk/tools/proguard/proguard-android.txt
```

Comment:
We can probably remove this whole file from git. Don't want to have references to specific file systems at any rate